### PR TITLE
Favor the use of PREFIX and BASE

### PR DESCRIPTION
**Pull request by gkellogg**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-turtle/pull/41.html" title="Last updated on Sep 18, 2023, 10:25 PM UTC (354b850)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-turtle/41/3aad884...354b850.html" title="Last updated on Sep 18, 2023, 10:25 PM UTC (354b850)">Diff</a>

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -1571,7 +1575,7 @@ <h3>Parsing Turtle in HTML</h3>
       of character data rather than a stream of UTF-8 encoded bytes.
       No decoding is necessary if the HTML document has already been parsed into DOM.
       Each <code>script</code> data block is considered to be it's own Turtle document.
-      <code>@prefix</code> and <code>@base</code> declarations in a Turtle data bloc
+      <code>PREFIX</code> and <code>BASE</code> declarations in a Turtle data bloc
```
Comment (domel) [inline spec/index.html:1578]: data bloc -> data block

File: spec/index.html [hunk 2]
```diff
@@ -1571,7 +1575,7 @@ <h3>Parsing Turtle in HTML</h3>
       of character data rather than a stream of UTF-8 encoded bytes.
       No decoding is necessary if the HTML document has already been parsed into DOM.
       Each <code>script</code> data block is considered to be it's own Turtle document.
-      <code>@prefix</code> and <code>@base</code> declarations in a Turtle data bloc
```
Comment (domel) [inline spec/index.html:1574]: data bloc -> data block

**Review by domel — COMMENTED** [review]
> No decoding is necessary if the HTML document has already been parsed into DOM. Each script data block is considered to be it's own Turtle document.
PREFIX and BASE declarations in a Turtle data bloc

Doesn't the above fragment suggest that the rule only applies to `PREFIX` and `BASE` and not to `@prefix` and `@base`?

File: spec/index.html [hunk 3]
```diff
@@ -400,7 +401,7 @@ <h3>IRIs</h3>
       @prefix p: <path/> .                    # prefix p: now stands for http://one.example/path/
       p:subject4 p:predicate4 p:object4 .     # prefixed name, e.g., http://one.example/path/subject4
 
-      @prefix : <http://another.example/> .    # empty prefix
+     PrEfIx : <http://another.example/>       # empty prefix
```
Comment (TallTed) [inline spec/index.html:404]:
```suggestion
      PrEfIx : <http://another.example/>       # empty prefix
```

File: spec/index.html [hunk 4]
```diff
@@ -1571,7 +1575,7 @@ <h3>Parsing Turtle in HTML</h3>
       of character data rather than a stream of UTF-8 encoded bytes.
       No decoding is necessary if the HTML document has already been parsed into DOM.
       Each <code>script</code> data block is considered to be it's own Turtle document.
```
Comment (TallTed) [inline spec/index.html:1577]:
```suggestion
      Each <code>script</code> data block is considered to be its own Turtle document.
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: spec/index.html (hunk 1 above)
Comment (gkellogg) [reply spec/index.html:1578]:
```suggestion
      <code>PREFIX</code> and <code>BASE</code> declarations in a Turtle data block
```

**Review by domel — APPROVED** [review]

**Pull request merged**